aws/resource_aws_apigatewayv2_integration.go

```diff
@@ -124,10 +124,9 @@ func resourceAwsApiGatewayV2Integration() *schema.Resource {
 				Optional: true,
 			},
 			"timeout_milliseconds": {
-				Type:         schema.TypeInt,
-				Optional:     true,
-				Default:      29000,
-				ValidateFunc: validation.IntBetween(50, 29000),
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
```

Comment:
Removed `ValidateFunc` as there's no way of making it dependent on the API type and the range 29000 to 30000 is invalid for WebSocket APIs.